source/docs/helpers.md

```diff
@@ -158,6 +165,13 @@ Loads JavaScript files. `path` can be an array or a string. If `path` isn't pref
 <%- js(['script.js', 'gallery.js']) %>
 // <script src="/script.js"></script>
 // <script src="/gallery.js"></script>
+
+<%- js({ src: 'script.js', integrity: 'foo' }) %>
```

Comment:
It is much more common to have `async` or `defer` attributes attached for `<script>`. We could add them to examples as well.